hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -239,10 +235,12 @@ public Response updateAdapter(final String adapterId, final Adapter adapter) {
     @Override
     public Response deleteAdapter(final String adapterId) {
         Optional<ProtocolAdapterWrapper> instance = protocolAdapterManager.getAdapterById(adapterId);
-        if (!instance.isPresent()) {
+        if (instance.isEmpty()) {
             return ApiErrorUtils.notFound("Adapter not found");
         }
-        logger.info("deleting adapter {}", adapterId);
+        if(logger.isTraceEnabled()){
```

Comment:
I would suggest debug as log level for the operations. They wont be called that often and they might be useful for debugging, wdyt?